src/peer-connection-wrapper.h

```diff
@@ -47,14 +49,19 @@ class PeerConnectionWrapper : public Napi::ObjectWrap<PeerConnectionWrapper>
   Napi::Value rtt(const Napi::CallbackInfo &info);
   Napi::Value getSelectedCandidatePair(const Napi::CallbackInfo &info);
 
-  // Close all existing DataChannels
+  // Close all existing Peer Connections
   static void CloseAll();
 
+  // Reset all Callbacks for existing Peer Connections
+  static void ResetCallbacksAll();
+
 private:
   static Napi::FunctionReference constructor;
   static std::unordered_set<PeerConnectionWrapper *> instances;
 
   void doClose();
+  void doDestroy();
+  void resetCallbacks();
```

Comment:
This one could be called `doResetCallbacks()` for consistency, to make it clear that it is not a JS-exposed method.